### PR TITLE
COMMERCE-4930 Fixed checkout process

### PR DESCRIPTION
**Pull request by alessiorendina**

### CODE REVIEW

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
ci:test:sf

**Review by dsitu — APPROVED** [review]

**Review by dsitu — CHANGES_REQUESTED** [review]
oh whoops looks like stable is failing because of 
```[echo] Detected build services changes. See above build services log for more information. ```
```[echo] Make sure to commit in all build services results.```

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:stop

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by marco-leo** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
ci:test:sf

**Review by dsitu — APPROVED** [review]

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderServiceImpl.java [hunk 1]
```diff
@@ -876,6 +877,22 @@ public CommerceOrder updateShippingAddress(
...
 	}
 
+	@Override
+	public CommerceOrder updateShippingMethod(
+			long commerceOrderId, long commerceShippingMethodId,
+			String shippingOptionName, CommerceContext commerceContext,
+			Locale locale)
+		throws PortalException {
+
+		_commerceOrderModelResourcePermission.check(
```
Comment (marco-leo) [inline modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderServiceImpl.java:887]: Use UPDATE no CHECKOUT_COMMERCE_ORDER

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
@Commando18 can you please rebase this PR?
thx

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Review by ivicac — APPROVED** [review]

**Comment by ivicac** [comment]
ci:forward

**Pull request closed without merging**